server/src/immich/controllers/person.controller.ts

```diff
@@ -28,6 +29,11 @@ function asStreamableFile({ stream, type, length }: ImmichReadStream) {
 export class PersonController {
   constructor(private service: PersonService) {}
 
+  @Post('/unmerge')
```

Comment:
What about delete /person/:id/asset/:assetId instead? And have it return a person response dto